driver/flambda_backend_args.ml

```diff
@@ -789,6 +807,7 @@ module Extra_params = struct
     | "reorder-blocks-random" ->
        set_int_option' Flambda_backend_flags.reorder_blocks_random
     | "heap-reduction-threshold" -> set_int' Flambda_backend_flags.heap_reduction_threshold
+    | "long-frames" -> set' Flambda_backend_flags.allow_long_frames
```

Comment:
Should we also have a "no" version
calling `clear'`?

Reply:
I should test it to double-check, but I thought that for `OCAMLPARAM` here `set'` would do the right thing for "long-frames=0", as it uses `Compenv.setter`. See for example "ocamlcfg" case above which I've used before with both 0 and 1. (We don't need `dno-asm-comments` below.)

Reply:
Very possibly, `"dno-asm-comments"`
(below) set me on the wrong track.